tests/acquia_lift.test

```diff
@@ -473,27 +481,35 @@ class AcquiaLiftWebTestConfig extends AcquiaLiftWebTestBase {
     );
   }
 
-  public function testLiftAgentAvailibility() {
-    // remove the account information
-    $lift_info = variable_get('acquia_lift_account_info', array());
+  public function testLiftAgentAvailability() {
+    // Fill all required information.
+    $this->configureAcquiaLiftAccount();
+    // Remove the account information.
     variable_del('acquia_lift_account_info');
```

Comment:
Very minor point, but why configure the account only to delete the variable straight away? :)